src/Apps/Collect2/Routes/Collections/Components/CollectionsGrid.tsx

```diff
@@ -58,7 +58,9 @@ export class CollectionsGrid extends Component<CollectionsGridProps> {
                   <Media at="xs">{index === 0 && <Separator />}</Media>
                   <Media greaterThan="xs">{index < 3 && <Separator />}</Media>
 
+                  {/* A empty space in initials to generate a blank avatar when there's no imageUrl*/}
```

Comment:
```suggestion
                  {/* An empty space in initials to generate a blank avatar when there's no imageUrl */}
```

Completely minor nit-pick for grammar & consistency of spacing. Feel free to disregard _or_ accept.